src/Feature/Demo/code/packages.config

```diff
@@ -6,17 +6,22 @@
   <package id="Microsoft.CodeDom.Providers.DotNetCompilerPlatform" version="1.0.3" targetFramework="net46" />
   <package id="Microsoft.Net.Compilers" version="1.3.2" targetFramework="net46" developmentDependency="true" />
   <package id="Microsoft.Web.Infrastructure" version="1.0.0.0" targetFramework="net46" />
-  <package id="Sitecore.Analytics.Automation.NoReferences" version="8.2.170614" targetFramework="net46" developmentDependency="true" />
-  <package id="Sitecore.Analytics.Core.NoReferences" version="8.2.170614" targetFramework="net46" developmentDependency="true" />
-  <package id="Sitecore.Analytics.Model.NoReferences" version="8.2.170614" targetFramework="net46" developmentDependency="true" />
-  <package id="Sitecore.Analytics.NoReferences" version="8.2.170614" targetFramework="net46" developmentDependency="true" />
-  <package id="Sitecore.Analytics.Outcome.NoReferences" version="8.2.170614" targetFramework="net46" developmentDependency="true" />
-  <package id="Sitecore.CES.DeviceDetection.NoReferences" version="8.2.170614" targetFramework="net46" developmentDependency="true" />
-  <package id="Sitecore.Cintel.NoReferences" version="8.2.170614" targetFramework="net46" developmentDependency="true" />
-  <package id="Sitecore.ExperienceEditor.NoReferences" version="8.2.170614" targetFramework="net46" developmentDependency="true" />
+  <package id="mongocsharpdriver" version="2.4.4" targetFramework="net46" />
```

Comment:
Is this a new requirement for Update 5?

Reply:
I'm not actually sure how it could be there. It is not needed. I will remove it